Makefile

```diff
@@ -106,7 +106,13 @@ $(ARTIFACTS):
 
 .PHONY: test
 test: ## Run tests
-	go test -v ./...
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./...
+
+.PHONY: test-cover
+test-cover: ## Run tests with code coverage and code generate  reports
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; mkdir -p /tmp/out; go test -v -coverprofile=/tmp/out/coverage.out ./... $(TEST_ARGS)
```

Comment:
Can we call the test target from above here? (please do it the same way as in the main repo, e.g. this also means using out/coverage.out instead the tmp folder)

Reply:
thanks. done